src/mainwindow.cc

```diff
@@ -119,6 +122,17 @@ MainWindow::MainWindow():
     if (sourceTypeSelection.isValid())
         sourceTypeComboBox->setCurrentIndex(sourceTypeSelection.toInt());
 
+    enableSystrayCheckButton->setChecked(config.value("systray/enabled", false).toBool());
+    startInSystrayCheckButton->setChecked(config.value("systray/startInTray", false).toBool());
+    closeToSystrayCheckButton->setChecked(config.value("systray/closeToTray", false).toBool());
+    
+    if(!enableSystrayCheckButton->isChecked()) {
+        startInSystrayCheckButton->setDisabled(true);
+        closeToSystrayCheckButton->setDisabled(true);
+    }
+
+    createTrayIcon();
```

Comment:
Could the systray icon be created/deleted only when it is needed?

Reply:
This ties to previous, should I make [Systray](https://github.com/lxqt/pavucontrol-qt/pull/93/files#diff-e53fc39ebc111e4e739351226712e953R44) into a real class in separate CO and move everything there?